src/site/content/en/fast/minify-css/index.md

```diff
@@ -0,0 +1,184 @@
+---
+layout: post
+title: Minify CSS
+author: demianrenzulli
+description: |
+  Learn how to minify CSS files, to improve performance, without affecting how the browser process the styles.
+web_lighthouse:
+- render-blocking-resources
+date: 2019-05-02
+---
+
+CSS files can contain unnecessary characters, such as comments, whitespaces, and indentation. 
+In production, these characters can be safely removed, to reduce file size without affecting how the browser processes the styles. This technique is called **minification**.
+
+## Loading unminified CSS
+
+Take a look at the following CSS block:
+
+```
+body {
+  font-family: "Benton Sans", "Helvetica Neue", helvetica, arial, sans-serif;
+  margin: 2em;
+}
+
+/* all titles need to have the same font, color and background */
+h1 {
+  font-style: italic;
+  color: #373fff;
+  background-color: #000000;
+}
+
+h2 {
+  font-style: italic;
+  color: #373fff;
+  background-color: #000000;
+}
+```
+
+This content is easy to read, at the cost of producing a larger than necessary file:
+
+- It uses spaces for indentation purposes and contains comments, which are ignored by the browser, so they can be removed.
+- The `<h1>` and `<h2>` elements have the same styles: instead of declaring them separately: "`h1 {...} h2 {...}`" they could be expressed as "`h1, h2{...}`".
+- The **background-color**, `#000000` could be expressed as just `#000`.
+
+After making these changes, you would obtain a more compact version of the same styles:
+
+```
+body{font-family:"Benton Sans","Helvetica Neue",helvetica,arial,sans-serif;margin:2em}h1,h2{font-style:italic;color:#373fff;background-color:#000}
+```
+
+You probably don't want to write CSS like that. Instead, you can write CSS as usual, and add a minification step to your build process. In this guide, you’ll learn how to do it by using a popular build tool: [Webpack](https://webpack.js.org/).
```

Comment:
```suggestion
You probably don't want to write CSS like that. Instead, you can write CSS as usual, and add a minification step to your build process. In this guide, you’ll learn how to do it by using a popular build tool: [webpack](https://webpack.js.org/).
```

Reply:
I glanced at the webpack docs and noticed they spell it all lowercase. I'll do a PR to update the site so we match their style. Also added it to our spelling wiki :) 

https://github.com/GoogleChrome/web.dev/wiki/Writing-style#spelling-and-capitalisation